core/src/banking_stage/consume_worker.rs

```diff
@@ -533,6 +543,21 @@ impl ConsumeWorkerTimingMetrics {
                 self.load_execute_us.swap(0, Ordering::Relaxed),
                 i64
             ),
+            (
+                "load_execute_us_min",
+                self.load_execute_us_min.swap(0, Ordering::Relaxed),
+                i64
+            ),
+            (
+                "load_execute_us_max",
+                self.load_execute_us_max.swap(0, Ordering::Relaxed),
+                i64
+            ),
+            (
+                "load_execute_us_mean",
+                self.load_execute_us.swap(0, Ordering::Relaxed)/self.count.swap(0, Ordering::Relaxed),
```

Comment:
You need to test the metrics you're adding. This will never be anything be 0 because we have already swapped `self.load_execute_us` to 0 above